doc/modules/kernel_ridge.rst

```diff
@@ -0,0 +1,57 @@
+.. _kernel_ridge:
+
+===========================
+Kernelized ridge regression
+===========================
+
+.. currentmodule:: sklearn.kernel_ridge
+
+Kernelized ridge regression (KRR) [M2012]_ combines ridge regression (linear 
+least squares plus l2-norm regularization) with the kernel trick. It thus
+learns a linear function in the space induced by the respective kernel and
+the data. For non-linear kernels, this corresponds to a non-linear
+function in the original space.
+
+The model learned by :class:`KernelRidge` is identical to support vector
```

Comment:
Maybe say "the form of the model" or "the parametrization of the model"? I feel like "the model learned" also includes the learned parameters, which will be different.